cyber/tools/cyber_recorder/spliter.cc

```diff
@@ -151,10 +151,11 @@ bool Spliter::Proc() {
       default: {
         AERROR << "this section should not be here, section type: "
                << section.type;
-        break;
+        return false;
```

Comment:
This changed the original logic, which only skips unknown sections and continues reading.

Reply:
I wonder why we should have unknown section at record file?

Reply:
Since the record structure may be upgraded in the future, spliter skips unknown sections and reads subsequent sections for ensure the post-compatibility.

Reply:
Make sense. thanks for your clarification!